### ISSUE TITLE
Log is incorrect in case of a group-specific Initial_* function

### ISSUE THREAD

**Issue by aplteam** [aplteam/Tester2#17]
This is how it looks like:
```
Test_Cache_001            ( 28/393) 
Function "Initial_Cache" found and successfully executed✓ Install a package with ...
```

This is how it should look like:
```
Function "Initial_Cache" found and successfully executed
Test_Cache_001            ( 28/393) ✓ Install a package with ...
```

**Comment by aplteam** [comment]
Fixed in 3.2.4

**Comment by aplteam** [comment]
Closed the wrong one...

**Comment by aplteam** [comment]
Fixed in 3.2.5
